src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -3063,6 +3113,19 @@ MillisecondsBetweenTimestamps(instr_time startTime, instr_time endTime)
 }
 
 
+/*
+ * MicrosecondsBetweenTimestamps is a helper to get the number of microseconds
+ * between timestamps when it is expected to be small enough to fit in a
+ * long.
+ */
+static long
+MicrosecondsBetweenTimestamps(instr_time startTime, instr_time endTime)
```

Comment:
This could go into `time_util.c` that we are adding with https://github.com/citusdata/citus/pull/4929/files#diff-dd0efb30ef4811fafbdbc33bf694f52dc887ed75bb4f2550810b571b48db6f95R1